dashboard/test/ui/features/authoredHints.feature

```diff
@@ -0,0 +1,98 @@
+Feature: Authored Hints
+
+Scenario: View Authored Hints
+  Given I am on "http://learn.code.org/s/allthethings/stage/6/puzzle/2?noautoplay=true"
+  And I rotate to landscape
+  And I wait to see "#x-close"
+  And I close the dialog
+  And I wait to see "#prompt-table"
+
+  # This level has a total of three authored hints
+  Then element "#lightbulb" is visible
+  And element "#hintCount" is visible
+  And element "#hintCount" has text "3"
+
+  When I press "prompt-table"
+  And I wait to see ".qtip"
+
+  Then element ".qtip" is visible
```

Comment:
I think this "Then" may be implied by the wait on line 16, maybe can be skipped (if this fails, subsequent lines fail too, right?)